config/locales/views/schools/user_tariffs.yml

```diff
@@ -49,6 +49,7 @@ en:
         introduction_school: Using your bill, please provide as much detail as you can about the consumption charges associated with this tariff
         introduction_school_group: Using your bill, please provide as much detail as you can about the consumption charges associated with this tariff
         introduction_site_settings: Please provide as much detail as you can about the consumption charges associated with this tariff
+        invalid_values_message: You have one or more invalid prices. Please add valid prices for all rates marked
```

Comment:
Maybe shorten this to "Please add valid prices for all marked rates", as this works slightly better when visiting the page initially as the user hasn't entered any prices yet.